### ISSUE TITLE
iPhone isses with selects and calendar picker

### ISSUE THREAD

**Issue by Desertsnowman** [CalderaWP/Caldera-Forms#608]
Issue with select fields on iphone not showing value.
example site: https://www.voyageafrique.com/contact/
possible to be an issue with parsley.

to replicate:
(1) On the "Pays" (Country) picker, I scroll to South Africa and select it, but it doesn't display in the field until I click in the next field down i.e. "Nombre d'adultes".

(2) The same thing happens when I enter a number in "Nombre d'adultes" - I have to scroll down and click in the next field i.e. "Nombre d'enfants".

(3) The calendar pop-up doesn't allow me to choose a date for "Periode du voyage" - this is where I can't go any further as I can't choose a date from the calendar and I can't manually fill in a date.

**Comment by ignusvermaak** [comment]
@Desertsnowman When is 1.3.6 expected and can you confirm this is definitely going to be fixed in that version?

**Comment by Desertsnowman** [comment]
@Shelob9 - please test can give me your feedback.

**Comment by Desertsnowman** [comment]
The wonderful thing about developing on mobile, is that pretty much every device supports HTML5.
I tried to use this to my advantage and detect mobile and revert to using the HTML5 `type="date"` field with no datepicker.

However. the HTML5 date field does not allow you to set the date format... SIGH!

So, I took an approach that on opening of the datepicker box, the field triggers a `blur` event. Thereby stopping the devices on screen keyboard from popping up and making the datepicker unreachable.

It's a good solution, however it does prevent you from being able to manually enter the date. I'm ok with this since its a date picker which by definition is to pick a date, not type it.

So closing via c1d0ada
